File: apps/booking-service-server/src/APIs/Room/Dtos/RoomUpdateInput.cs
namespace BookingService.APIs.Dtos;

public class RoomUpdateInput
{
    public List<string>? Bookings { get; set; }

    public int? Capacity { get; set; }

    public DateTime? CreatedAt { get; set; }

    public string? Features { get; set; }

    public string? Id { get; set; }

    public string? RoomNumber { get; set; }

    public DateTime? UpdatedAt { get; set; }
}

File: apps/hotel-reservation-service-admin/src/api/room/RoomUpdateInput.ts
import { BookingUpdateManyWithoutRoomsInput } from "./BookingUpdateManyWithoutRoomsInput";

export type RoomUpdateInput = {
  bookings?: BookingUpdateManyWithoutRoomsInput;
  capacity?: number | null;
  features?: string | null;
  roomType?: string | null;
};
